CODEOWNERS

```diff
@@ -278,8 +278,7 @@
 /scripts/elf_helper.py                    @andrewboie
 /scripts/sanity_chk/expr_parser.py        @nashif
 /scripts/gen_app_partitions.py            @andrewboie
```

Comment:
> Create new GLOBAL PROPERTY PM_IMAGES

This is stated in the commit message. Are you creating this property? I'm only seeing PM_IMAGES being read in this commit, not set.

Also, there is a problem with the language here:

> The new file contains all the other file generation (header and kconfig)
> It takes the output yaml as input.

Reply:
PM_IMAGES is only set (appended) in fw-nrfconnect-nrf, but there's no reason it can't be set in this repo as well. Maybe I should "define" it?